spec/web_player_factory_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "web_player_factory"
+require "tictactoe/game_type_options"
+
+RSpec.describe WebPlayerFactory do
```

Comment:
Is this class fully tested? seems like only one of the four cases are actually tested.